### ISSUE TITLE
Keyboard controls to 'fight'

### ISSUE THREAD

**Issue by feildmaster** [feildmaster/UnderScript#3]
Use keyboard keys (possibly 1-4) to select a card on the board to attack another card.

Maybe add a way to select a card from your hand as well

**Comment by nwawrzyniak** [comment]
Maybe try to mirror the board on your key board as default to give the controls a natural feel?
That would be something like:
Q-W-E-R: enemy monster zones 1-4 (maybe shifted more into the middle like E-R-T-Y or R-T-Y-U) [Y instead of Z on QWERTZ layout]
A-S-D-F: ally monster zones 1-4 (same as above, maybe D-F-G-H)
Z-X-C-V-B-N-M: hand cards 1-7 [Y instead of Z on QWERTZ layout]

**Comment by feildmaster** [comment]
Nah, that's not really intuitive to actually use. You shouldn't need that many keys for this type of thing.
